blog/authors.yml

```diff
@@ -19,4 +19,8 @@ vasvi:
   title: Open Source Contributor
   url: https://github.com/vasvi-sood
   image_url: https://github.com/vasvi-sood.png
-
+travishathaway:
```

Comment:
I'm not sure why the diff is showing Travis's author info and not yours, but if you look at the comment at the top of this file, your author info should actually go beneath Travis's, since your author ID starts with a V and his with a T.

Reply:
Ah, I think it was because I was looking at a specific commit and not the entire branch. Still, if you could put your author info at the bottom, so it's in alphabetical order, that'd be great.